app/Decsys/Data/Entities/Mongo/Webhook.cs

```diff
@@ -1,10 +1,11 @@
 using Decsys.Models.Webhooks;
+using MongoDB.Bson;
 
 namespace Decsys.Data.Entities.Mongo;
 
 public class Webhook
```

Comment:
The TriggerCriteria property here should no longer be a list, as we have changed how the trigger criteria model works